.gitignore

```
@@ -0,0 +1 @@
+**/.DS_Store
```

```suggestion
**/.DS_Store

```

It is good hygiene to have a newline at the end of each line, and you can see that the last line in this file does not have one and github is complaining.  I won't highlight all the occurrences, but you should review and see whether you want to update this.  You should be able to configure your IDE to include a newline at the end of each file